Berlin/BerOrOct555.xml

```diff
@@ -0,0 +1,218 @@
+                        </dimensions>
+                      </extent>
+
+                      <foliation>
+                         <list>
+                            <item xml:id="f.1r">
+                               <locus from="1r" to="120r"/>
+                               Foliation in pencil in the upper right corner.
+                            </item>
+                         </list>
+                      </foliation>
+                      </supportDesc>
+
+                      <layoutDesc>
+                            <layout columns="2" writtenLines="17"/>
+                            </layoutDesc>
+                      </objectDesc>
+
+                      <handDesc>
+                         <handNote xml:id="h1" script="Ethiopic">
+                           <locus from="6ra" to="118ra"/>
+                           <seg type="script">Late fifteenth-century script.</seg>
+                           <seg type="ink">Black, red.</seg>
+                           <desc>Careful, well trained hand.</desc>
+                         </handNote>
+                       </handDesc>
+
+                       <additions>
+                          <list>
+                             <item xml:id="a1">
+                                <locus target="#2v"/>
+                                <desc type="Invocation">Short invocation.</desc>
+                                <q xml:lang="gez">
+                                </q>
+                             </item>
+
+                             <item xml:id="a2">
+                                <locus target="#118rb"/>
+                                <desc type="OwnershipNote">The beginning of the note was erased, no names inserted.</desc>
+                                <q xml:lang="gez">
+                                </q>
+                             </item>
+                           </list>
+                        </additions>
+
+                       <bindingDesc>
+                         <binding xml:id="binding">
+                           <decoNote xml:id="b1">Ethiopian. Two wooden boards covered with reddish
+                             brown blind-tooled leather. Blind-tooled mitred turn-ins, textile inlays.
+                             </decoNote>
+                            <decoNote xml:id="b2" type="bindingMaterial">
+                            <material key="wood"/>
+                            <material key="leather"/>
+                            <material key="textile"/>
+                            </decoNote>
+                            <decoNote xml:id="b3" type="SewingStations">4</decoNote>
+                            <decoNote xml:id= "b4" type="Endbands">Head and tail bands in dark brown leather.</decoNote>
+                            <decoNote xml:id="b5" type="Other">
+                              <term key="leafStringMark"></term> inserted in the upper corner of ff. 6, 16, 20, 35, 53, 65, 78, 82, 86, 94
```

Comment:
I would use here a locus element to avoid having to decide for ff. fols etc. `<locus target="#6 #16 #20...">`

Reply:
ahhhhh :)